### PR TITLE
Harmonize publisher visits

### PR DESCRIPTION
**Pull request by mrose17**
Fixes https://github.com/brave/browser-laptop/issues/3830

test plan:

open two windows. in one window, put the payments pane; in the other two tabs on the same publisher but different URLs. observer the visits and duration counts in the first window and then switch back and forth between tabs (waiting one or two minutes between each switch). the visits should stay constant and the duration should increase.

### CODE REVIEW

File: app/ledger.js [hunk 1]
```diff
@@ -470,7 +470,8 @@ var updatePublisherInfo = () => {
     var entries = []
 
     underscore.keys(publishers[publisher]).forEach((location) => {
```
Comment (diracdeltas) [inline app/ledger.js]: can just do `underscore.each(publishers, ...)`

**Comment by diracdeltas** [comment]
please include a test plan in the commit message
@bridiver should also audit this

File: app/ledger.js (hunk 1 above)
Comment (mrose17) [inline app/ledger.js]: +1

**Comment by mrose17** [comment]
i will add a test plan. i had a brief conversation with @bridiver about this...

File: app/ledger.js (hunk 1 above)
Comment (mrose17) [inline app/ledger.js]: in this case, no... because in the function i need access to both the key and value, not just the value...

**Comment by mrose17** [comment]
fixes https://github.com/brave/browser-laptop/issues/3830

File: app/ledger.js (hunk 1 above)
Comment (diracdeltas) [inline app/ledger.js]: both key and value are accessible http://underscorejs.org/#each

**Comment by bridiver** [comment]
the eventStore changes look good to me. Discussed the definition of a visit with @mrose17 and that sounds ok too. @diracdeltas I think you are in a better position than I am to review the ledger code.

**Pull request merged**